config.ru

```diff
@@ -5,12 +5,15 @@
 # This file is used by Rack-based servers to start the application.
 
 require ::File.expand_path("../config/environment",  __FILE__)
-require ::File.expand_path("../lib/unicorn_killer",  __FILE__)
 require ::File.expand_path("../lib/rack/internet_explorer_version", __FILE__)
 
 # Kill unicorn workers really aggressively (at 300mb)
 if defined?(Unicorn)
-  use UnicornKiller::Oom, 300 * 1024
+  require "unicorn/worker_killer"
```

Comment:
Not necessarily, since the `defined?(Unicorn)` makes unicorn optional with no changes besides the Gemfile.